spec/player_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rspec'
+require './lib/deck'
+require './lib/card'
+require './lib/player'
+require 'pry'
+
+RSpec.describe Player do
+  describe 'Iteration 2: player'
+  it 'creates a player' do
+    card1 = Card.new(:diamond, 'Queen', 12)
+    card2 = Card.new(:spade, '3', 3) 
+    card3 = Card.new(:heart, 'Ace', 14)
+    deck = Deck.new([card1, card2, card3])
+    player = Player.new('Clarisa', deck)
+
+    expect(player).to be_an_instance_of(Player)
+    expect(player.name).to eq('Clarisa')
+    expect(player.deck).to eq(deck)
+  end
+  
+  it 'creates a lose condition' do
+    card1 = Card.new(:diamond, 'Queen', 12)
+    card2 = Card.new(:spade, '3', 3) 
+    card3 = Card.new(:heart, 'Ace', 14)
+    deck = Deck.new([card1, card2, card3])
+    player = Player.new('Clarisa', deck)
+
+    expect(player.has_lost?).to eq(false)
+    player.deck.remove_card
+    expect(player.has_lost?).to eq(false)
+    player.deck.remove_card
+    expect(player.has_lost?).to eq(false)
+    player.deck.remove_card
+    expect(player.has_lost?).to eq(true)
```

Comment:
I like how this tests that `has_lost?` returns true **only** when the deck has enough cards removed.